pnpm-lock.yaml

```diff
@@ -2863,14 +2863,14 @@ packages:
       react-dom: 18.2.0(react@18.2.0)
     dev: false
 
-  /@floating-ui/react@0.26.9(react-dom@18.2.0)(react@18.2.0):
-    resolution: {integrity: sha512-p86wynZJVEkEq2BBjY/8p2g3biQ6TlgT4o/3KgFKyTWoJLU1GZ8wpctwRqtkEl2tseYA+kw7dBAIDFcednfI5w==}
+  /@floating-ui/react@0.26.3(react-dom@18.2.0)(react@18.2.0):
+    resolution: {integrity: sha512-iKH8WRR0L/nLiM6qavFZxkyegIZRMxGnM9aKEc71M4wRlUNkgTamjPsOQXy11oZbDOH37MiTbk/nAPn9M2+shA==}
```

Comment:
Actually, should we pin the version to `0.26.9` instead of `0.26.3`, since that what we were using so far?

Reply:
I guess there is no science behind the exact version here. Would it be time-consuming to find out which exact version broke our tests?

Reply:
It looks like the problem cannot be reproduced anymore with latest versions of floating-ui deps. I think last week I may have ended up in a broken lockfile state with incompatible versions of transitive floating-ui deps (or maybe they fixed something in 0.26.10).

So then I would say there is no need to pin `@floating-ui/react` because it does not affect which transitive dep versions get installed. We can release 3.7.4 without it

And in 3.8.0, we can bump minimum required version to `^0.26.10`.